pkg/operator/controller.go

```diff
@@ -172,6 +170,23 @@ func (c *Controller) sync() error {
 			return err
 		}
 	}
+	statusChanged := !reflect.DeepEqual(prevCR.Status, cr.Status)
+	if statusChanged {
+		glog.Infof("object changed: %s (status=%t)", util.ObjectInfo(cr), statusChanged)
+
+		client, err := regopset.NewForConfig(c.kubeconfig)
+		if err != nil {
+			return err
+		}
+
+		_, err = client.ImageregistryV1().Configs().UpdateStatus(cr)
+		if err != nil {
+			if !errors.IsConflict(err) {
```

Comment:
I'm concerned that since may have just updated spec above, this is very likely to get a conflict.  Can we do something to avoid that scenario?